### DISCUSSION TITLE
Apple iPadOS16,iOS 16 and macOS Ventura 13

### DISCUSSION THREAD

**Discussion by taassolucoes** [InnovateAsterisk/Browser-Phone#396] [Q&A]
After updating Apple products, as soon as you answer a call originating from the Web Browser (Safari or Chrome) on the Softphone.
Video windows "disappear", on Android and Windows 10/11 it works perfectly.
Before answering, my preview video appears

**Comment by InnovateAsterisk** [comment]
I see that: https://webcamtests.com/
Also has the same issue in iOS safari and chrome. I’ll do some research on how to fix this.

**Comment by taassolucoes** [comment]
It works perfectly.
Even testing at https://sipjs.com/ runs flawlessly.
In the previous version it worked perfectly.
From the side that receives the call, I receive the video normally.
On the WEB side when I change the cameras the self appears
